### DISCUSSION TITLE
Welcome to NameIt Discussions!

### DISCUSSION THREAD

**Discussion by 5axes** [5axes/NameIt#1] [Announcements]
# 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ImagineerNL** [comment]
Hi there, 
It's me, the guy who asked for the original feature to be able to add (text) identifiers on the print bed.  You were so kind and quick to inplemwnt it in the original plugin. 
So when you notified me of building a separate plugin to name-it.... well, I just had to 'get involved'. 

Even though i am originally an IT guy, i have zero programming knowledge. But i do have a mind that can make sense of things :)
